main.cpp

```diff
@@ -81,53 +87,56 @@ struct <#structName1#>                                //4
 
 struct U
 {
-    float <#name1#> { 0 }, <#name2#> { 0 };
-    <#returnType#> <#memberFunction#>(<#type name#>* <#updatedValue#>)      //12
+    float x { 0 }, y { 0 };
+    float multiply(float* updatedValue)      //12
     {
-        
+        std::cout << "U's x value: " << this->x << std::endl;
+        this->x = *updatedValue;
+        std::cout << "U's x updated value: " << this->x << std::endl;
```

Comment:
What must we always do before we use a pointer?
- [ ] revise

Reply:
check for nullptr